### PR TITLE
Simplified Readme to include neccessary details

### PR DESCRIPTION
**Pull request by nainardev**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [ ] Include a change request file using `$ yarn change`

#### Description of changes

(give an overview)

#### Focus areas to test

(optional)

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/fluentui/pull/12465)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,32 +41,21 @@ Please see the [wiki](https://github.com/microsoft/fluentui/wiki).
...
-```
+```sh
+npx create-react-app my-app
 cd my-app
 npm start
 ```
 
-This scaffold uses the [`just`](https://github.com/microsoft/just) build library. It is very flexible and requires no "eject" script to allow for customization on its configurations.
-
-If you prefer to use `create-react-app`, [follow these instructions](https://github.com/microsoft/fluentui/wiki/Getting-Started-with-UI-Fabric#create-react-app).
```
Comment (aneeshack4) [inline README.md]: I think we would want to keep this... there's no harm in linking to our own wiki's creact-react-app page.

File: README.md [hunk 2]
```diff
@@ -41,32 +41,21 @@ Please see the [wiki](https://github.com/microsoft/fluentui/wiki).
...
 
-```
-npm init uifabric
-```
-
-It'll prompt you for a project name. If you choose to create a project called `my-app`, you can start working on the project like this:
-
-```
+```sh
+npx create-react-app my-app
```
Comment (aneeshack4) [inline README.md]: why npx instead?

Comment (nainardev) [reply README.md]: Installing create-react-app globally is officially discouraged. npx installs and executes the latest version & is the recommended way

File: README.md (hunk 1 above)
Comment (nainardev) [reply README.md]: Almost all of React users and starters use create-react-app, so it drastically simplifies installation instructions in README

File: README.md [hunk 3]
```diff
@@ -41,32 +41,21 @@ Please see the [wiki](https://github.com/microsoft/fluentui/wiki).
...
 cd my-app
 npm start
```
Comment (dzearing) [inline README.md]: Before running `npm start`, we should move the instruction below to add fluentui up here:

```tsx
cd my-app
npm install @fluentui/react
npm start
```

File: README.md (hunk 2 above)
Comment (dzearing) [reply README.md]: That makes sense.

File: README.md (hunk 1 above)
Comment (dzearing) [reply README.md]: I think that makes sense as well. There a number of unaddressed considerations here instructing to use the `uifabric` starter app (mainly docs), and those go away by recommending CRA, which has a lot of usage and documentation available.

File: README.md [hunk 4]
```diff
@@ -41,32 +41,21 @@ Please see the [wiki](https://github.com/microsoft/fluentui/wiki).
...
 
 ### Integrating in your project
 
-How to integrate components into your project depends heavily on your setup. The recommended setup is to use a bundler such as [Webpack](https://webpack.js.org/) which can resolve NPM package imports in your code and bundle only the specific things you import.
-
-Within an npm project, you should install the package and save it as a dependency:
-
-```
+```sh
 npm i @fluentui/react
```
Comment (dzearing) [inline README.md]: This is that part which should move up before running `npm start`.

**Comment by dzearing** [comment]
Thanks @nainardev for the updates! @joschect will make similar updates on the website documentation.

**Comment by nainardev** [comment]
My first PR :party:

**Pull request merged**